PrismaDB-QueryAST/Result/DataColumn.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using PrismaDB.QueryAST.DDL;
+using PrismaDB.QueryAST.DML;
+
+namespace PrismaDB.QueryAST.Result
+{
+    public class DataColumn
```

Comment:
Maybe better to call `DataColumnHeader` or `ResultColumnHeader`. I thought you are organizing data by columns initially when I saw the class name.